react-accordion-component/my-app/src/Accordion.jsx

```diff
@@ -0,0 +1,41 @@
+import './Accordion.css';
+import { useState } from 'react';
+
+export default function Accordion({ listOfTopics }) {
+  const [current, setCurrent] = useState('');
+
+  function handleCustomClick(e) {
+    if (e.target.textContent === current) {
+      setCurrent('');
+    } else {
+      setCurrent(e.target.textContent);
+    }
+  }
+
```

Comment:
I'd suggest not passing whole event object up the component tree. Instead, write your handlers to only expect the value itself.

Reply:
Fixed